pyannote/database/registry.py

```diff
@@ -0,0 +1,194 @@
+from enum import Enum
+from pathlib import Path
+from typing import Dict, List, Set, Text, Tuple, Type, Union
+from .custom import create_protocol, get_init, get_custom_protocol_class_name
+from .database import Database
+from .config import get_database_yml
```

Comment:
This should also become a method of `Registry`

Reply:
I suppose this is about `get_database_yml`, I think the function is no longer needed in its current state.

But currently, pyannote.database first tries to find a database.yml in this order : first the cwd, then the environment variable, and finally in `~/.pyannote/database.yml`.

Should this behaviour be kept ? And in this case, should we still try to load only one of them, or all of them (then, with what "override" behaviour and what order) ?

Reply:
We should keep this behavior and put it in `Registry.__init__` with the following differences:

* it should load all of them in order (`~/.pyannote/database.yml`, then `cwd/database.yml`, then `os.environ["PYANNOTE_DATABASE_CONFIG"]) , allowing overrides, but letting the user know that overrides happen.
* it should not complain if it cannot find any of these `database.yml` files

Reply:
Should the default behaviour of `registry.load_databases(...)` also be "_override existing protocols, with a warning_" (`OverrideType.WARN_OVERRIDE`)? Currently the default parameter value is "_keep existing protocols, with a warning_" (`OverrideType.WARN_KEEP`).

I'm updating the README and I suppose it would be clearer if files loaded at startup were loaded using the default value of `load_database`'s `allow_override: OverrideType` parameter (if the default behaviour was consistent).

(And we can choose either of them, if we decide to use with `OverrideType.KEEP_WARNING`, we'd just have to reverse the startup loading order.)

Reply:
Yes. Thanks. Go ahead!